src/sentry/static/sentry/app/views/discover/conditions/condition.tsx

```diff
@@ -210,33 +210,34 @@ export default class ConditionRow extends React.Component<
   };
 
   render() {
+    const {disabled, value} = this.props;
```

Comment:
@mgaeta confused why this component is changing

Reply:
This was linting from the first round of debugging.

Reply:
@mgaeta so are you removing it? I'm still confused